src/Servers/Kestrel/Transport.Quic/src/Internal/QuicStreamContext.cs

```diff
@@ -498,18 +498,17 @@ public override void Abort(ConnectionAbortedException abortReason)
         var resolvedErrorCode = _error ?? 0;
         QuicLog.StreamAbort(_log, this, resolvedErrorCode, abortReason.Message);
```

Comment:
Can we produce a different log (or no log?) when `_stream` is null which would imply the stream had already stopped?

Reply:
I've made `Abort` exit immediately if `_stream` is null. There is nothing for it to do: no stream to abort and pipes are completed in dispose.